packages/sdk-cli/src/models/localRelay/launch.test.ts

```diff
@@ -1,18 +1,30 @@
 import { join } from 'path';
-import { ChildProcess } from 'child_process';
+import * as child_process from 'child_process';
 import { createWriteStream, promises as fsPromises } from 'fs';
 
 import { launch } from './launch';
 import { RELAY_DIRECTORY_NAME } from './const';
 
+import { mockStreamWithEventEmit } from './index.test';
+
+// Is there a more idiomatic way to do this? Just using .spy() without mocking
```

Comment:
`jest.spyOn(child_process, 'spawn').mockImplementation()` doesn't work?

Reply:
I get a "Cannot set property spawn of [object Object] which has only a getter" error, which is something babel-related I think. I followed this https://stackoverflow.com/a/53209819/6539857

Reply:
Having read that, I think what you have here is fine. I can't say I'd ran into that before, it's a new one on me too!